src/main/scala/no/ndla/articleapi/service/converters/contentbrowser/LenkeConverterModule.scala

```diff
@@ -29,12 +30,21 @@ trait LenkeConverterModule {
     }
 
     def convertLink(cont: ContentBrowser): (String, Seq[RequiredLibrary], Seq[String]) = {
-      val url = extractService.getNodeEmbedUrl(cont.get("nid")).get
+      val embedMeta = extractService.getNodeEmbedMeta(cont.get("nid")) match {
```

Comment:
Skulle dette være med? (Siden pull requesten heter noe annet :) )
Og, hvis ja, hva er det egentlig du fikser? Commit-meldingen antyder at vi skal hente ut metainformasjon for noe som "ikke finnes" som gir litt lite mening i mitt hode :)